Makefile

```diff
@@ -361,16 +361,25 @@ supplier_stake: ## Stake tokens for the supplier specified (must specify the APP
 	poktrolld --home=$(POKTROLLD_HOME) tx supplier stake-supplier 1000upokt "$(SERVICES)" --keyring-backend test --from $(SUPPLIER) --node $(POCKET_NODE)
 
 .PHONY: supplier1_stake
-supplier1_stake: ## Stake supplier1
-	SUPPLIER=supplier1 SERVICES="anvil;http://anvil:8547,svc1;http://localhost:8081" make supplier_stake
+supplier1_stake: ## Stake supplier1 (also staked in genesis)
+	# TODO_IMPROVE: consolidate supplier1 staking; why/where is supplier1
+	# being staked other than this make target?
+	# Supplier1 seems to be staked at some point during localnet startup.
```

Comment:
Currently supplier1 and app1 stake state is being applied in genesis. However, this does not include the corresponding public keys which are for most practical operations. Resolving this situation requires requires the respective actor to submit a tx of some kind in order to put their public key on-chain.

```suggestion
```